package.json

```diff
@@ -19,7 +19,7 @@
       "url": "https://github.com/Kicchu02"
     },
     {
-      "name": "Maithry V Pappu",
+      "name": "Maithry V. Pappu",
```

Comment:
Why?

Reply:
> Why?
I believe this happened because of the pulled changes from your branch was reverted, hence the package.json was switched the previous state.
Pls let me know how do I rectify this one.

Reply:
Just remove the dot.